src/url-utils.ts

```diff
@@ -1,10 +1,29 @@
 import { resolve } from 'url';
 
-const getUrl = (base: string, projectName?: string): string => {
+const getUrl = (
+  base: string,
+  projectName?: string,
+  namePrefix?: string,
+  tags?: Array<string>,
+): string => {
+  const url = resolve(base, './client/features');
+  const params = new URLSearchParams(url);
```

Comment:
you should not append `url` to the URLSearchParams. 

This ends up creating:
`http://some-url?http://someUrl=`

Simply replace this line with:
`const params = new URLSearchParams();`

Reply:
Uh you are right. Probably left this after some testing. Will fix in next commit.